.rubocop.yml

```diff
@@ -27,3 +27,8 @@ Style/FrozenStringLiteralComment:
 
 Metrics/BlockLength:
   Max: 35
+
+Style/ClassAndModuleChildren:
+  EnforcedStyle: compact
+  Exclude:
+    - 'config/**/*'
```

Comment:
an empty line is forgotten, hey, you just need to config text editor to add an empty line at a file end